### ISSUE TITLE
Problems displaying the settings section

### ISSUE THREAD

**Issue by denel-manilov** [DEgITx/rats-search#4]
System:
```
Distributor ID:	Ubuntu
Release:	17.10
Codename:	artful
Kernel: 4.13.0-32-generic
Window Manager: xfwm4
Desktop environment: xfce
```
App version: **0.3.0** AppImage

Screenshot:
![selection_006](https://user-images.githubusercontent.com/23494247/35900947-f281b926-0bed-11e8-850a-fd65aa3a969f.png)

**Comment by DEgITx** [comment]
What version is this (I mean dev, rpm, AppImage or other)?

**Comment by denel-manilov** [comment]
@DEgITx 
sorry forgot
App version: **0.3.0**

**Comment by DEgITx** [comment]
still same problem in last version or something different?

**Comment by denel-manilov** [comment]
Tested:
App version: **0.9.0 AppImage**
App version: **0.9.0 deb**
Problem displaying the settings section solved

I found other display problems:
**About -> Changeling** page is empty

**Comment by DEgITx** [comment]
This is another issue

**Comment by denel-manilov** [comment]
App version: **0.9.0 AppImage**

I do not open a new issue, because I do not see any problems in the application, but I got some errors in the terminal:
```
upnp error Error: timeout
    at Timeout._onTimeout (/tmp/.mount_rats-seUxyT9/app/resources/app.asar/node_modules/nat-upnp/lib/nat-upnp/client.js:187:14)
    at ontimeout (timers.js:386:14)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
upnp error Error: timeout
    at Timeout._onTimeout (/tmp/.mount_rats-seUxyT9/app/resources/app.asar/node_modules/nat-upnp/lib/nat-upnp/client.js:187:14)
    at ontimeout (timers.js:386:14)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
upnp error Error: timeout
    at Timeout._onTimeout (/tmp/.mount_rats-seUxyT9/app/resources/app.asar/node_modules/nat-upnp/lib/nat-upnp/client.js:187:14)
    at ontimeout (timers.js:386:14)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
Update for version 0.9.0 is not available (latest version: 0.9.0, downgrade is disallowed.
```

**Comment by DEgITx** [comment]
This is "ok" errors) I will probably disable this ones

**Comment by denel-manilov** [comment]
I was confused about this line: 
```
Update for version 0.9.0 is not available (latest version: 0.9.0, downgrade is disallowed.
```

**Comment by DEgITx** [comment]
This is ok message too, this means that you are using latest version of application
